contribute/backend/style-guide.md

```diff
@@ -27,7 +27,7 @@ make lint-go
 
 We value clean and readable code, that is loosely coupled and covered by unit tests. This makes it easier to collaborate and maintain the code.
 
-Tests must use the standard library, `testing`. For assertions, prefer using [testify](https://github.com/stretchr/testify).
+Tests must use the standard library, `testing`. For assertions, we prefer using [testify](https://github.com/stretchr/testify).
```

Comment:
Avoid "we": https://developers.google.com/style/pronouns

```suggestion
Tests must use the standard library, `testing`. For assertions, prefer using [testify](https://github.com/stretchr/testify).
```

Reply:
Line 28 clearly starts with "we value clean and readable code." I was making the code consistent.

Reply:
That's a good point. Although consistency is valuable, I think the use of we should be removed from line 28 rather than added in line 30.